app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class attribute, meaning it is shared among all instances of `Animal`. If you want each instance to have its own `alive` list, consider making it an instance attribute by moving it inside the `__init__` method.